content-audit/api/src/main/java/org/commonjava/propulsor/content/audit/model/FileEventType.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2011-2018 Red Hat, Inc. (https://github.com/Commonjava/propulsor)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.commonjava.propulsor.content.audit.model;
+
+public enum FileEventType
```

Comment:
I'd suggest removing what is after line 24~. Basically, we don't need them. See https://github.com/Commonjava/galley/blob/master/api/src/main/java/org/commonjava/maven/galley/model/TransferOperation.java

Reply:
And not need the string value in the quote.

Reply:
k, let's keep consistent.